docs/ref/modules/management/README.md

```diff
@@ -1 +1,91 @@
-# Management API
+# Management REST API
+
+## Introduction
+
+The REST API allows the remote management of the Wazuh infrastructure such as:
+ - CRUD operations for agents
+ - CRUD operations for groups
+ - Agent's assignments to groups
+ - CRUD operations for security entities(Users, Roles, Rules and Policies)
+ - Restarting the node(s) or agent(s)
+
+ ## Configuration
+
+| Option | Description | Mandatory | Default |
+|--------|-------------|-----------|---------|
+| host | The host address for the communications API |  | localhost |
+| post | The port number for the communications API |  | 55000 |
+| drop_privileges | Whether to drop privileges after starting the API | | true |
+| max_upload_size | The maximum upload size in bytes | | 10485760 |
+| jwt_expiration_timeout | The expiration timeout for JWT in seconds | | 900 |
+| rbac_mode | The role-based access control mode. Accepted values: white, black | | white |
```

Comment:
```suggestion
```
The option will be removed as part of https://github.com/wazuh/wazuh/issues/27706.